runtime/doc/starting.txt

```diff
@@ -471,7 +471,18 @@ accordingly.  Vim proceeds in this order:
 	The |-V| argument can be used to display or log what happens next,
 	useful for debugging the initializations.
 
-3. Execute Ex commands, from environment variables and/or files
+3. Source early plugin files				    *early-plugins*
+	This does the same as >
+	    source early/**/*.vim
+<	The result is that all directories in the default |'runtimepath'| will be
+	searched for the "early" sub-directory and all files ending in ".vim"
+	will be sourced (in alphabetical order per directory), also in
+	subdirectories.
+	Loading plugins won't be done when:
```

Comment:
I must sound like the biggest pedant of all time, but "if" might be better in this context... sorry :sweat: